recipes/stactools/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "stactools" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stactools-{{ version }}.tar.gz
+  sha256: 691dc0038d3e709462748747d2e39c2f369cf16d5cd890faba0025e4424b4991
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+  - stac = stactools.cli:cmd
```

Comment:
Is this correct? It is `stactools.cli.cli:run_cli` in [setup.cfg](https://github.com/stac-utils/stactools/blob/1f565873604a337d376314b1b8e38241802a94e8/setup.cfg#L62). The best is to add a test to ensure the entry point works as expected.

Reply:
Corrected